### PR TITLE
[ft-164112378]-election-results

### PR DESCRIPTION
**Pull request by sengayire**
**What does this PR do?**
- a user should be able to view the result of the election by political office

***Description of Task to be completed?***
 - a user should view the results from the election for each candidate 

**How should this be manually tested?**
- This can be tested by cloning locally this repo and use postman for the test or visit this link:  https://politico-andela.herokuapp.com/

### CODE REVIEW

File: server/controllers/office.js [hunk 1]
```diff
@@ -68,6 +68,46 @@ const office = {
...
 
+  // election results
+
+  async results(req, res) {
+    try {
+      const { id } = req.params;
+      // Check if office not exists
+      let findOffice = officeQueries.fetchOffices;
+
+      findOffice += ' WHERE id = $1';
```
Comment (FredDominant) [inline server/controllers/office.js]: Why are you doing this?

File: server/routes/office.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const officeRoute = express.Router();
 officeRoute.post('/create', validate, office.officeTable);
 officeRoute.post('/', validate, office.create);
 // officeRoute.get('/', office.getAllOffices);
```
Comment (FredDominant) [inline server/routes/office.js]: remove commented code

File: server/routes/office.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const officeRoute = express.Router();
...
-// officeRoute.get('/:id', office.getOneOffice);
+officeRoute.get('/:id/result', office.results);
 // officeRoute.delete('/:id', office.deleteOne);
```
Comment (FredDominant) [inline server/routes/office.js]: here too

**Review by FredDominant — CHANGES_REQUESTED** [review]
Address the comments then merge.

File: server/routes/office.js (hunk 3 above)
Comment (sengayire) [reply server/routes/office.js]: marge after apply feedback

Comment (sengayire) [reply server/routes/office.js]: marge after apply feedback
```suggestion
// officeRoute.delete('/:id', office.deleteOne);
```

**Pull request merged**